latiss/transmission_sensor/README.md

```diff
@@ -0,0 +1,9 @@
+This directory contains sensor QE curve for the LATISS sensor, ITL-3800C-068.
+
+Data Processing
+---------------
+The input FITS file was downloaded from https://srs.slac.stanford.edu/DataCatalog/dataset.jsp?dataset=40352803&experiment=LSST-CAMERA which contains the Brookhaven National Lab Test Stand 3 (TS3) measurements of the sensor QE.
+
+This was then converted to the calib format by running the script `python $OBS_LSST_DIR/python/lsst/obs/lsst/script/rewrite_latiss_qe_file.py`.
+
+Both the original FITS and output ecsv files have been committed to this repo.
```

Comment:
Maybe confirm that it's the ECSV version that is the one used by the calibration system?

Reply:
Are you worried the other files there might be picked up?  Should they go in a different directory?  But the calibration ecsv file is the one that is used, and it is the only one with the correct name.  (On jira I have a script that shows that the files are working as intended).

Reply:
Is there a downside to an explicit comment? It might be that it would be best to put the original source files somewhere else. It does occur to me as well that it might be nice to add provenance to the ECSV file indicating where the data came from.

Reply:
Which part?

Reply:
What explicit comment?  Where?
And for original source files, I'm happy to hear suggestions as to where they should go.
Finally, for provenance, you mean just maybe pointing to the README in a comment?  Or something else?

Reply:
Explicit comment: "The FITS file is not used further by the calibration system and is here for provenance purposes."

I was thinking that the rewrite script could add the file name to the metadata of the ECSV file. I'm not sure of the best keyword but maybe something like "PROVSRC = 'ITL-3800C-068_QE.fits'" ? (it's not easily possibly for it to be a path including `$OBS_LSST_DATA_DIR`). At least a header like this gives an explicit linkage between the .fits file in the package and the .ecsv package.

As for location, I imagine it doesn't really matter. Naming is hard. We'd end up with a parallel tree in the package somewhere. I don't think we stored the TS8 QE files in the package that Simon worked on. In the general case though, I don't think we are going to want ~200 FITS files in here for ~200 ecsv files -- if the QE curves change we will have even more FITS files all in the same directory. When I think about hundreds of ECSV and FITS files in here it does make me want to put the FITS files in a different place. A `_src` subdir?

Reply:
Adding a keyword is a good idea, although we've already been inconsistent on this.  I think my suggestion here is going to be to choose any reasonable key name to add the source file name to the newly added ECSV files.  As this ticket already has edits on the LATISS `transmission_filter` files, those should be updated to that same keyword (it's just `SOURCE` now).

The FITS file is safe from being picked up as we restrict the inputs in this package to be one of `{.ecsv, .yaml, json}`, so there's no reason to move them out of the way now.  I think moving them to a `_src` directory makes sense, but I don't have a clear idea of where that directory should live (do we have one top level `_src` containing all of our sources, do we have one for each instrument, or do we have one for each calibration type), so I propose adding a "clean up" ticket that can handle that decision, do the moves, and enforce whatever keyword choices onto the existing files without making this ticket any bigger.

Reply:
So should I just add a `SOURCE` key to each of these ecsv files?  That makes sense to me.

Reply:
Okay. At some point we should make the rewrite script do that.

Reply:
Yup, that script is on this ticket (in obs_lsst), so that's where I was going to put it.  It's simply a question of the name of the key.

Reply:
`SOURCE` works for me.